src/api/main.py

```diff
@@ -85,6 +94,64 @@ def run_rwd(lat, lon):
         return error_response(exc.message, stack_trace)
 
 
+@app.route('/rwd-nhd/<lat>/<lon>', methods=['GET'])
+def run_rwd_nhd(lat, lon):
+    """
+    Runs RWD NHD for lat/lon and returns the GeoJSON
+    for the computed watershed boundary, outlet point
+    and snapped point.
+    """
+    snapping = request.args.get('snapping', '1')
+    maximum_snap_distance = request.args.get('maximum_snap_distance', '10000')
+    simplify = str(request.args.get('simplify', 0.0001))
+
+    num_processors = '1'
+    data_path = '/opt/rwd-data'
```

Comment:
Yeah, I think we should keep the nhd data separate from the drb data in case of accidental name collisions.